src/utils/index.js

```diff
@@ -14,3 +14,13 @@ export const getQueryString = query =>
       return parameters;
     }, {})
   );
+
+export const arePropTypesIdentical = (nextProps, currentProps) => {
```

Comment:
Would rename `arePropTypesIdentical` -> `arePropsIdentical` since this is not about the types.

Alternatively, could name it more generic - `shallowCompare`

Reply:
it is about the types. Check the function : 

```
if (typeof nextProps[key] !== typeof currentProps[key]) {
```

it has a limited use, but in some cases when a component is getting only functions in props we don't want to update it when nothing really changed.

Reply:
Oh, right. I completely overread that 😄

I think it would be a better idea to fix it by not passing in arrow functions from the parent. The current solution will have false positives when the function actually changed